props/saber_fett263_buttons.h

```diff
@@ -4733,7 +4733,7 @@ SaberFett263Buttons() : PropBase() {}
     }
 #elif defined(FETT263_SAY_BATTERY_PERCENT)
     sound_library_.SayBatteryPercent();
-#elif defined(FETT263_SAY_BATTERY_VOLTAGE)
+#elif defined(FETT263_SAY_BATTERY_VOLTS)
```

Comment:
typo in define

Reply:
One PR per feature would be better, but I'll let it slide this time.